package.json

```diff
@@ -50,7 +50,8 @@
         "onCommand:aws.github",
         "onView:aws.cdk.explorer",
         "onCommand:aws.refreshCdkExplorer",
-        "onCommand:aws.aboutToolkit"
+        "onCommand:aws.aboutToolkit",
+        "workspaceContains:**/*template.{yaml,yml,json}"
```

Comment:
shouldn't we wait on this until startup cost is reduced? This will eager-start the Toolkit extension

Reply:
Yes was just testing this will remove for now